### ISSUE TITLE
"Open PowerShell and start hacking with docker or compose" message should be disabled

### ISSUE THREAD

**Issue by Rainmaker52** [docker/for-win#4708]
### Expected behavior
It should be a user choice whether or not the message "Open PowerShell and start hacking with docker or compose" is displayed on start.

Every time I boot my machine, Docker reminds me that it is installed. I know it is installed. It is the only application that finds itself important enough to remind me of that fact every time I login.

This message should be optional and should be (optionally) disabled.

### Actual behavior
The message is displayed every time. Because it is a "toast" message, it also causes sound to play. This is very annoying.

### Information
Since Windows 1903, it is possible to disable messages from the Windows notification settings. However, I believe any application should have at least an option to disable these types of messages.

### Steps to reproduce the behavior
1. Install
2. Reboot

**Comment by ArthurHNL** [comment]
I can not stress this enough. I have docker set up to start whenever I log in so getting the following notification

![image](https://user-images.githubusercontent.com/31919595/67237427-873a9a00-f44b-11e9-9245-eec0f7e0903a.png)

every time I sign in is just useless and annoys me.

**Comment by sudoscientist** [comment]
I don't have a solution, but I just wanted to let you know that I share your pain. This notification is completely obnoxious and pops up every morning.

**Comment by MartinWilkerson** [comment]
Remember you can always tell Windows to hide the notifications - in the notification drawer there should be a gear icon you can click on the specific notification to block Docker For Windows from showing notifications. 

Of course this probably blocks _all_ notifications from DfW, but it'll do until they decide Docker isn't the most amazing thing I'm going to use my computer for today.

**Comment by escalonn** [comment]
According to Docker Forums this problem has been around for over three years. Frankly ridiculous.

**Comment by Rainmaker52** [comment]
/lifecycle frozen

I guess if it's already there for a few years, and more people than just me are annoyed by it, the bot shouldn't close this one.

**Comment by ArthurHNL** [comment]
Indeed. I'm wondering if the Docker team is even checking this issue list...

**Comment by ArthurHNL** [comment]
Note that this issue still has the `lifecycle/stale` label so it will still be auto-closed in 30 days...

**Comment by Rainmaker52** [comment]
> 
> 
> Indeed. I'm wondering if the Docker team is even checking this issue list...

Honestly, I think they are. But look at this list. Over half of the issues reported are just GUIDs, with no explanation or text whatsoever. The other half is people is completely misunderstanding the purpose of Docker, asking questions like where the start button is, now that they've started it.

I guess this is only a minor issue, with a workaround available. It's an issue nonetheless, so in my opinion, it should be fixed.

**Comment by vitalyster** [comment]
Any updates on this?
